CMakeLists.txt

```diff
@@ -100,6 +100,15 @@ option(TBB_NO_APPCONTAINER "Apply /APPCONTAINER:NO (for testing binaries for Win
 option(TBB4PY_BUILD "Enable tbb4py build" OFF)
 option(TBB_CPF "Enable preview features of the library" OFF)
 option(TBB_FIND_PACKAGE "Enable search for external oneTBB using find_package instead of build from sources" OFF)
+option(TBB_TBBBIND_BUILD "Enable building TBBbind libraries" ON)
+
+if (APPLE)
+    set(TBB_TBBBIND_BUILD OFF)
+endif()
```

Comment:
It look strange, we are adding option to enable tbbbind build but turn off on some platforms. What is the option usage model? It seems that it does not respect the user setting.

Reply:
It was decided to rework option semantic and rename it to `TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH `